test/e2e/all_in_one_test.go

```diff
@@ -407,3 +428,52 @@ data:
   password: %s`, adminPasswordSecretName, adminPasswordB64)
 	return adminPassword, adminPasswordSecretYAML, nil
 }
+
+// exposeAdminAPI will override the KONG_ADMIN_LISTEN for the cluster's proxy to expose the
+// Admin API via a service. Some deployments only expose this on localhost by default as there's
+// no authentication, so note that this is only for testing environment purposes.
+func exposeAdminAPI(ctx context.Context, t *testing.T, env environments.Environment) *corev1.Service {
```

Comment:
Even we expose admin API service, we do not have much interface to use in dbless mode. do we still need this function ?

Reply:
It is already being used in a test case here:

https://github.com/Kong/kubernetes-ingress-controller/pull/1816/files#diff-d7850eda46771ac1eb42d866bb557fd98bdea88b6bcfba6b26b10cc975a40856R331

This is used in when the `TestDeployAllInOneEnterpriseDBLESS` test runs.